### DISCUSSION TITLE
平时都在用别的B站助手，但每隔几个月回来看下，这边何时解决宽屏+定位功能

### DISCUSSION THREAD

**Discussion by CHINAupup** [the1812/Bilibili-Evolved#3236] [功能建议 / Ideas]
一直没对策，在油猴里做备用助手。主要是各家助手，谁的夜间模式时不时就失效，就转用他家。谁也不想麻烦，综合利弊我也想固定用一家B站助手。
就目前而言，此助手唯一缺点就只有标题所述的了。无他可定居使用。
平时助手主要用这一些功能，夜间模式、宽屏、定位、顶栏的动态视频展示
(超级依赖夜间模式💕)

**Comment by the1812** [comment]
- 夜间模式更新速度是减弱了的
- 由我维护的几个相关功能均已停止开发, 包括`默认播放器模式`, `播放器置顶`, `播放器定位`.
- 有个由 @timongh 开发的 `视频页默认定位` 的功能可以替代 `播放器定位`.
